boards/posix/nrf52_bsim/cmsis.c

```diff
@@ -0,0 +1,62 @@
+#include <stdint.h>
```

Comment:
What kind of license header should be put here?

Reply:
Apache 2
Given that you are copying some code from an Apache 2.0 file with Oticon's copyright, you should take that header, and then add an extra copyright line for your company if you want